test/uk/gov/hmrc/platforminitiatives/controllers/PlatformInitiativesControllerSpec.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.platforminitiatives.controllers
+
+import org.mockito.MockitoSugar
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.mvc.Results
+
+class PlatformInitiativesControllerSpec extends AnyWordSpec with Matchers with Results with MockitoSugar {
+
+  "Platform Initiatives controller" should {
+    "have the correct url set up for the initiatives list" in {
+      uk.gov.hmrc.platforminitiatives.controllers.routes.PlatformInitiativesController.allInitiatives()
+        .url mustBe "/initiatives"
```

Comment:
you can also call the controller functions, mocking the services, and check response code/body